test/data/articles/rss.xml

```diff
@@ -0,0 +1,4547 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
do we really need all 4000 lines of this?

Reply:
Not all xml files can be distilled. So to reproduce I took the original xml file.